profiles/apparmor/apparmor.go

```diff
@@ -68,6 +71,10 @@ func macroExists(m string) bool {
 // InstallDefault generates a default profile in a temp directory determined by
 // os.TempDir(), then loads the profile into the kernel using 'apparmor_parser'.
 func InstallDefault(name string) error {
+	if !aaparser.IsAvailable() {
```

Comment:
I'm not sure this is where it makes sense to check -- it really seems like `generateDefault()` is overloaded/should not concern itself with asking the `apparmor_parser` for its version directly. On the other hand, an early return instead of attempting to execute the apparmor template also makes sense.

Overall, I think we're hitting a lot of code smells here, and some restructuring might be in order.